src/journal/views.py

```diff
@@ -998,11 +998,11 @@ def issue_galley(request, issue_id, delete=False):
                 issue_galley.replace_file(uploaded_file)
             except models.IssueGalley.DoesNotExist:
                 file_obj = files.save_file(
-                        request,
-                        uploaded_file,
-                        label=issue.issue_title,
-                        public=False
-                        *(models.IssueGalley.FILES_PATH, issue.pk)
+                    request,
+                    uploaded_file,
+                    label=issue.issue_title,
+                    public=False,
+                    path_parts=(models.IssueGalley.FILES_PATH, issue.pk)
```

Comment:
All the other file functions are using the `*(path_parts)` pattern. I think we should either change all of them or none of them for consistency

Reply:
I can change them all, or if you prefer it as it is can you work out why it wasn't working?

Reply:
It is not working because of the missing comma on the previous line (classic python gotcha). I think  you are right, it is easier to read with the kwarg than with the unpacking